mdoc/Makefile

```diff
@@ -48,7 +48,7 @@ MULTI-UNIFIED = Test/DocTest-DropNS-unified.dll Test/DocTest-DropNS-unified-mult
 
 DIFF = diff -rup
 DIFF_QUIET = diff --brief
-ifeq ($(PLATFORM), win32)
+ifeq ($(OS), Windows_NT)
```

Comment:
were you able to get the makefile building on windows? if so that's *awesome*! Can you update [the readme](https://github.com/mono/api-doc-tools#cli) to include any specifics on how you got this working (including what version of `make` you installed)?

Reply:
ok that makes sense